persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1504,6 +1519,14 @@ escape s =
     go ('"':xs) = "\"\"" ++ go xs
     go (x:xs) = x : go xs
 
+-- | Escape a database name to be included on a query.
+escapeDBName :: String -> String
```

Comment:
basically does the same thing as the above method, but with `String`s instead of `Text`.  In reviewing this, it makes me want to just refactor the DB method to use `Text` instead of `String`s.